### ISSUE TITLE
AG WG Review: 3.3.5 Help (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#953]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 3.3.5 Help, as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 3.3.5 Help to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-3-3-5-help-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [3.3.5 Help](https://www.w3.org/TR/WCAG22/#help) SC text in WCAG 2.2
* [Intent section of Understanding 3.3.5 Help](https://www.w3.org/WAI/WCAG22/Understanding/help#intent)

**Comment by maryjom** [comment]
The notes for this SC need more work.

editorial: 
- Note 1: It would be clearer that the link is to the WCAG definition if you said "As noted in the WCAG definition of [context-sensitive help](https://www.w3.org/TR/WCAG22/#dfn-context-sensitive-help),..."

substantive:
- Note 1: It is unclear what is meant by "or other information provided by the system". What system? By the user agent or the operating system? The definition of "context-sensitive help" does not say anything about a system. So if this note is trying to add more information than what the WCAG definition contains, it should be in a separate sentence where it's clear that outside of that definition, for non-web documents that context sensitive help may be provided by some other means and describe it more clearly. I've added the WCAG definition here for reference: 
>help text that provides information related to the function currently being performed
>Note: Clear labels can act as context-sensitive help.
- Note 2: The sentence "This success criterion readily applies to player/viewer software application." doesn't fit in with non-web documents. A player/viewer is software, not a document. Suggest removing that sentence or saying instead that this note does not apply to viewers/players of non-web documents which are considered non-web software. 
- Note 2: The 2nd and and 3rd sentences are a bit confusing. The 2nd sentence says that the SC may be problematic to apply, but doesn't explain why.  
Then the third sentence gives examples where this SC could be supported without any transition information. Suggest adding a sentence to say something like: However, some document formats have authoring tools that support adding context-sensitive help to user input elements included in the document.  ...and then give the examples which I found quite useful.
